tests/unit/store/settings.spec.js

```diff
@@ -300,21 +300,21 @@ describe("store/settings", () => {
       "settings/set_customer_details",
       add_customer
     ); /*  (Save ID) selected */
-    let current_customerids = store.getters["settings/customer_account_ids"];
+    const current_customerids = store.state.settings.customer_account_ids;
     current_customerids.push(store.getters["settings/customer_details"]);
     store.commit(
       "settings/set_customer_details",
       current_customerids
     ); /*  (SaveChanges) selected */
-    expect(store.getters["settings/customer_account_ids"].length).toEqual(
+    expect(store.state.settings.customer_account_ids).toHaveLength(
       3
     ); /*  assert the number of customer_account_ids to three */
   });
-  test("Store a multiple customer detail with multiple user ids can be assigned to the customer_ids array an 'Edit Customer' ==>  PopopUpDialog (SaveID) ==> SettingsForm (SaveChanges) ", () => {
+  test("Given the store has multiple customer details, When the user updates one Customer details, Then validate the updates is reflecting in the store", () => {
```

Comment:
same comment as previous test: the user is not updating anything here (no clicks or typing), a vuex mutation is being committed--please update the When section of the test title to better reflect what is being tested